client/src/pages/teds/charts/PrctIpccPublicationsByCountry.tsx

```diff
@@ -49,11 +49,11 @@ export default function PrctIpccReferencesByCountry() {
   }
 
   const series = (data?.aggregations?.by_countries?.buckets ?? []).map((item: { key: string, doc_count: number }) => ({
-    color: item.key === 'FR' ? '#cc0000' : '#808080',
-    name: item.key,
+    color: item.key === 'France' ? '#cc0000' : '#808080',
     y: item.doc_count / data.hits.total.value * 100,
+    number: item.doc_count
   }));
-  const categories = series.map((country) => country.name);
+  const categories = series.map((country) => country.name +'<br>'+'('+country.number+')');
```

Comment:
`country.name` is not supposed to exist anymore, `country.key` might be  the correct one to use.

Reply:
Plus, prefer Template literals reather than string concatenation : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#syntax